flytekit/common/translator.py

```diff
@@ -123,6 +123,7 @@ def get_serializable_task(
         interface=entity.interface,
         custom=entity.get_custom(settings),
         container=entity.get_container(settings),
+        task_type_version=entity.task_type_version,
```

Comment:
have we tried to serialize any old examples? just to make sure they continue to work? the house price predictor is a good example maybe?

Reply:
the default task type version should still be 0 so existing dynamic array tasks should be unaffected. only map tasks increment the version